ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -1,36 +1,32 @@
 import { ethErrors, serializeError } from 'eth-rpc-errors';
 import React, { useCallback, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
+
 import {
   Button,
   ButtonSize,
   ButtonVariant,
   IconName,
 } from '../../../../../components/component-library';
+import { ConfirmAlertModal } from '../../../../../components/app/alert-system/confirm-alert-modal';
 import { Footer as PageFooter } from '../../../../../components/multichain/pages/page';
+import { doesAddressRequireLedgerHidConnection } from '../../../../../selectors';
 import { useI18nContext } from '../../../../../hooks/useI18nContext';
 ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
 import { useMMIConfirmations } from '../../../../../hooks/useMMIConfirmations';
 ///: END:ONLY_INCLUDE_IF
-
-import { doesAddressRequireLedgerHidConnection } from '../../../../../selectors';
 import {
   rejectPendingApproval,
   resolvePendingApproval,
 } from '../../../../../store/actions';
+import useAlerts from '../../../../../hooks/useAlerts';
 import { confirmSelector } from '../../../selectors';
 import { getConfirmationSender } from '../utils';
-import useAlerts from '../../../../../hooks/useAlerts';
-import { ConfirmAlertModal } from '../../../../../components/app/alert-system/confirm-alert-modal';
-import { Severity } from '../../../../../helpers/constants/design-system';
-import { Alert } from '../../../../../ducks/confirm-alerts/confirm-alerts';
-import useIsDangerButton from './useIsDangerButton';
 
-function getIconName(hasUnconfirmedAlerts: boolean): IconName {
-  return hasUnconfirmedAlerts ? IconName.SecuritySearch : IconName.Danger;
-}
+const getIconName = (hasUnconfirmedAlerts: boolean): IconName =>
```

Comment:
As discussed before, I'd vote to use the `function` syntax for traditional top level functions.

For the components themselves, I appreciate it's personal preference whether we go `const` or `function`.